package.json

```diff
@@ -98,6 +99,7 @@
     "babel-loader": "^6.2.4",
     "babel-preset-es2015": "^6.6.0",
     "bcapp-pattern-lab": "git@github.com:bigcommerce-labs/bcapp-pattern-lab.git",
+    "bower": "^1.8.2",
```

Comment:
do we need bower?

Reply:
Yes apparently ng-common depends on it but it is not defined as a dependency there, so `npm install` fails without this.